apis/stackgres/v1/sgcluster.yaml

```diff
@@ -0,0 +1,4155 @@
+openapi: "3.0.2"
```

Comment:
Do we need to keep these yaml files around?

Reply:
That's actually added in https://github.com/vshn/appcat-comp-functions/pull/33, but I screwed up the rebase.

That one is actually necessary, but a can clean ups some of them. Will do.